pgtap/withPoints/withPointsDD/edge_cases/issue_979.pg

```diff
@@ -20,53 +20,17 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 BEGIN;
 
-SELECT PLAN(6);
```

Comment:
Please do not change pgtap tests at this time, put it back as it was ...
Of course pgtap will fail, it is expected.